pkg/storage/mysql/mysqlstore_test.go

```diff
@@ -331,13 +339,107 @@ func TestSQLDBStore(t *testing.T) {
 		require.NoError(t, err)
 		verifyItr(t, itr, 1, "")
 
-		itr, err = store.Query(`""`)
-		require.Error(t, err)
+		itr, e := store.Query(`""`)
+		require.Error(t, e)
 		require.Nil(t, itr)
-		require.Contains(t, err.Error(), "failed to query rows")
+		require.Contains(t, e.Error(), "failed to query rows")
 	})
+	t.Run("Successfully query using index", func(t *testing.T) {
+		err := createIndex(store, "`key`", storeName)
+		require.NoError(t, err)
+		//nolint: gosec
+		itr, err := store.Query("SELECT * FROM " + storeName + "" +
+			" USE INDEX (" + testIndexName + ") WHERE `key` = 'abc_124'")
+		require.NoError(t, err)
+
+		ok, e := itr.Next()
+		require.NoError(t, e)
+		require.True(t, ok)
+
+		value, e := itr.Value()
+		require.NoError(t, e)
+		require.Equal(t, "val-for-abc_124", string(value))
```

Comment:
Same question about using `e` instead of `err`

Reply:
same as above